tests/commons/j2/test_j2_functions.py

```diff
@@ -182,11 +182,13 @@ def test_string(self):
                 "'\"'^^xsd:string",
                 "bad %s format" % type_name,
             )
+            # fmt: off
             self.assertEqual(
                 xsd_fmt(">'<", type_name, quote="'"),
-                "'''>'<'''^^xsd:string",
+                "'''>\\'<'''^^xsd:string",
```

Comment:
still wrong -- as suspected we are now introducing double escape --> line 110 is same as 105

expectation should be: `"'''>\'<'''^^xsd:string"`